PyComPWA.cpp

```diff
@@ -300,27 +354,27 @@ PYBIND11_MODULE(ui, m) {
   py::class_<ComPWA::Tools::IntensityComponent>(m, "IntensityComponent");
 
   py::class_<ComPWA::Physics::IntensityBuilderXML>(m, "IntensityBuilderXML")
-      .def(py::init([](const std::string &filename, ComPWA::ParticleList partL,
-                       ComPWA::Kinematics &kin,
-                       const ComPWA::EventCollection &PhspSample) {
-             boost::property_tree::ptree pt;
-             boost::property_tree::xml_parser::read_xml(filename, pt);
-             auto it = pt.find("Intensity");
-             if (it != pt.not_found()) {
-               ComPWA::Physics::IntensityBuilderXML Builder(
-                   partL, kin, it->second, PhspSample);
-               return Builder;
-             } else {
-               throw ComPWA::BadConfig("pycompwa::IntensityBuilderXML(): "
-                                       "Intensity tag not found in xml file!");
-             }
-           }),
-           "Create an intensity and a helicity kinematics from a xml file. The "
-           "file "
-           "should contain a particle list, and a kinematics and intensity "
-           "section.",
-           py::arg("xml_filename"), py::arg("particle_list"),
-           py::arg("kinematics"), py::arg("phsp_sample"))
+      .def(
+          py::init([](const std::string &filename, ComPWA::ParticleList partL,
+                      ComPWA::Kinematics &kin,
+                      const ComPWA::EventCollection &PhspSample) {
+            boost::property_tree::ptree pt;
+            boost::property_tree::xml_parser::read_xml(filename, pt);
+            auto it = pt.find("Intensity");
+            if (it != pt.not_found()) {
+              ComPWA::Physics::IntensityBuilderXML Builder(
+                  partL, kin, it->second, PhspSample);
+              return Builder;
+            } else {
+              throw ComPWA::BadConfig("pycompwa::IntensityBuilderXML(): "
+                                      "Intensity tag not found in xml file!");
+            }
+          }),
+          "Create an intensity and a helicity kinematics from a xml file. The "
```

Comment:
I dont get why this is reformatted again

Reply:
Has to do with refitting this chained builder pattern after I changed the description a bit, I think

Reply:
because I see no changes

Reply:
Could also have to do with the editorconfig, I'll have a look

Reply:
Strange, with `clang-format` 6.0.0 and the editorconfig for in this PR, I keep ending up in the same formatting. Could you give it a try?

Reply:
I thought we are using clang-format 8

Reply:
Not sure, can't find whether we concluded anything formal on that, only this https://github.com/ComPWA/pycompwa/pull/40#issuecomment-576727173

Reply:
I have clang-format 8 as well, but 6 is the default on ubuntu 18

Reply:
Hm, clang-format 8 [seems to make it worse](https://github.com/redeboer/pycompwa/compare/redeboer:feat-data...redeboer:feat-data-clang-format-8) (temporary link)

Reply:
Hmm, it seems to be related to this docstring (second argument). Depending on how you format this. It will intend everything or not